torch/autograd/variable.py

```diff
@@ -278,10 +278,12 @@ def btrifact(self, info=None, pivot=True):
             return super(Variable, self).btrifact(pivot=pivot)
 
     def resize(self, *sizes):
+        warnings.warn("non-inplace resize is deprecated", DeprecationWarning)
```

Comment:
I think `DeprecationWarning`s are not shown by default. We usually just use `warnings.warn(msg)` which defaults to `UserWarning`